### DISCUSSION TITLE
目前支持Mistral-7B-Instruct系列，比如Mistral-7B-Instruct-v0.3吗？

### DISCUSSION THREAD

**Discussion by gonghuan2024** [chatchat-space/Langchain-Chatchat#4227] [Q&A] [answered]
请问目前支持Mistral-7B-Instruct系列，比如Mistral-7B-Instruct-v0.3吗？

支持的话要如何使用？要修改哪些配置或者文件？
不支持的话后续会考虑支持吗？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
支持的，现在都是自己选模型了，ollama等框架都支持，转为OpenAI格式接入chatchat就行了

**Reply by gonghuan2024** [reply]
我用的是chatchat0.2.10的版本，是huggingface上下载的Mistral-7B-Instruct-v0.1，然后更新了chatchat的配置文件，看着模型是启动了，但是LLM问答的时候，返回很慢很慢，然后还返回的很多重复的消息，请问下知道这个是什么原因吗？
[response_1718872059547.txt](https://github.com/user-attachments/files/15948861/response_1718872059547.txt)
返回信息：
: ping - 2024-06-20 12:31:51.646113

: ping - 2024-06-20 12:32:06.646930

: ping - 2024-06-20 12:32:21.646989

: ping - 2024-06-20 12:32:36.647662

: ping - 2024-06-20 12:32:51.647410

: ping - 2024-06-20 12:33:06.647273

: ping - 2024-06-20 12:33:21.647396

: ping - 2024-06-20 12:33:36.647551

: ping - 2024-06-20 12:33:51.647430

: ping - 2024-06-20 12:34:06.647801

: ping - 2024-06-20 12:34:21.648827

: ping - 2024-06-20 12:34:36.649249

: ping - 2024-06-20 12:34:51.649317

: ping - 2024-06-20 12:35:06.649345

: ping - 2024-06-20 12:35:21.649178

: ping - 2024-06-20 12:35:36.649656

: ping - 2024-06-20 12:35:51.649821

: ping - 2024-06-20 12:36:06.650838

: ping - 2024-06-20 12:36:21.651899

: ping - 2024-06-20 12:36:36.652953

: ping - 2024-06-20 12:36:51.653064

: ping - 2024-06-20 12:37:06.653362

: ping - 2024-06-20 12:37:21.653509

: ping - 2024-06-20 12:37:36.653390

: ping - 2024-06-20 12:37:51.653694

: ping - 2024-06-20 12:38:06.654731

: ping - 2024-06-20 12:38:21.655186

: ping - 2024-06-20 12:38:36.655917

: ping - 2024-06-20 12:38:51.656569

: ping - 2024-06-20 12:39:06.656934

: ping - 2024-06-20 12:39:21.657103

: ping - 2024-06-20 12:39:36.657046

: ping - 2024-06-20 12:39:51.657223

: ping - 2024-06-20 12:40:06.657022

: ping - 2024-06-20 12:40:21.657665

: ping - 2024-06-20 12:40:36.657909

: ping - 2024-06-20 12:40:51.658695

: ping - 2024-06-20 12:41:06.659184

: ping - 2024-06-20 12:41:21.659910

: ping - 2024-06-20 12:41:36.660840

: ping - 2024-06-20 12:41:51.661348

: ping - 2024-06-20 12:42:06.661632

: ping - 2024-06-20 12:42:21.661064

: ping - 2024-06-20 12:42:36.661088

: ping - 2024-06-20 12:42:51.661580

: ping - 2024-06-20 12:43:06.661113

: ping - 2024-06-20 12:43:21.661440

: ping - 2024-06-20 12:43:36.661907

: ping - 2024-06-20 12:43:51.662451

: ping - 2024-06-20 12:44:06.662767

: ping - 2024-06-20 12:44:21.663890

: ping - 2024-06-20 12:44:36.664856

: ping - 2024-06-20 12:44:51.665417

: ping - 2024-06-20 12:45:06.665835

: ping - 2024-06-20 12:45:21.666728

: ping - 2024-06-20 12:45:36.667817

: ping - 2024-06-20 12:45:51.668615

: ping - 2024-06-20 12:46:06.668909

: ping - 2024-06-20 12:46:21.669146

: ping - 2024-06-20 12:46:36.669761

: ping - 2024-06-20 12:46:51.670294

: ping - 2024-06-20 12:47:06.669989

: ping - 2024-06-20 12:47:21.670231

: ping - 2024-06-20 12:47:36.670072

: ping - 2024-06-20 12:47:51.670365

: ping - 2024-06-20 12:48:06.669967

: ping - 2024-06-20 12:48:21.670865

: ping - 2024-06-20 12:48:36.671414

: ping - 2024-06-20 12:48:51.671544

: ping - 2024-06-20 12:49:06.671896

: ping - 2024-06-20 12:49:21.672752

: ping - 2024-06-20 12:49:36.673682

: ping - 2024-06-20 12:49:51.674637

: ping - 2024-06-20 12:50:06.674597

: ping - 2024-06-20 12:50:21.674603

: ping - 2024-06-20 12:50:36.674449

: ping - 2024-06-20 12:50:51.674135

: ping - 2024-06-20 12:51:06.674174

: ping - 2024-06-20 12:51:21.674797

: ping - 2024-06-20 12:51:36.675598

: ping - 2024-06-20 12:51:51.675590

: ping - 2024-06-20 12:52:06.675369

data: {"text": "怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒发冲冠 [INST] 怒发冲冠 [/INST] 怒怒发冲发冲冠怒怒发冲冠 [INST]怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲冠 [INST] 怒怒发冲冠 [/INST] 怒怒发冲", "message_id": "e64e08a6860a473881f723b96eb791b4"}
